fast64_internal/oot/oot_collision.py

```diff
@@ -382,20 +386,23 @@ def ootCameraPosToC(camPos):
 		str(camPos.jfifID) + ', ' +\
 		str(camPos.unknown) + ' },\n'
 
-def ootCameraEntryToC(camPos, camData, camPosIndex):
+def ootCameraEntryToC(camPos, camData, camPosIndex, hasPosData):
 	return " ".join((
 		"{",
 		camPos.camSType + ',',
 		('3' if camPos.hasPositionData else '0') + ',',
-		('&' + camData.camPositionsName() + '[' + str(camPosIndex) + ']'),
+		(('&' + camData.camPositionsName() + '[' + str(camPosIndex) + ']') if hasPosData else "NULL"),
```

Comment:
`camPos.hasPositionData` and `hasPosData` are the same here afaict, probably don't need to add the `hasPosData` argument